### PR TITLE
MM-30300: Add coalesce to get null content as empty string on FileInfo store

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Add coalesce to get null content as empty string on FileInfo store

#### Ticket Link
[MM-30300](https://mattermost.atlassian.net/browse/MM-30300)

#### Release Note
Not needed because is fixing a problem created in master

### CODE REVIEW

**Review by reflog — APPROVED** [review]

File: store/sqlstore/file_info_store.go [hunk 1]
```diff
@@ -230,7 +260,8 @@ func (fs SqlFileInfoStore) GetForUser(userId string) ([]*model.FileInfo, error)
 
 	if _, err := dbmap.Select(&infos,
 		`SELECT
-				*
+				*,
+				COALESCE(Content, '') as Content
```
Comment (streamer45) [inline store/sqlstore/file_info_store.go:264]: Should we also use `fs.queryFields` here?

Comment (jespino) [reply store/sqlstore/file_info_store.go:264]: sure :facepalm:

Comment (jespino) [reply store/sqlstore/file_info_store.go]: Done

**Review by streamer45 — APPROVED** [review]
Looks good :+1:

**Review by lindy65 — APPROVED** [review]
Thanks @jespino - tested on an HA spinwick with various images (as well as the ones that weren't loading in Inktober channel) and this is fixed - all images are loading. 👍

**Comment by jespino** [comment]
@lindy65 Great, do you mind to test it again on community once it is deployed?

**Pull request merged**